changelog.d/20220208_114626_aaschaer_gcs_roles.rst

```diff
@@ -0,0 +1,14 @@
+..
+.. A new scriv changelog fragment
+..
+.. Add one or more items to the list below describing the change in clear, concise terms.
+..
+.. Leave the ":pr:`...`" text alone. When you open a pull request, GitHub Actions will
+.. automatically replace it when the PR is merged.
+..
+
+* Add role methods to ``GCSClient`` (:pr:`NUMBER`)
+    ** ``GCSClient.get_role_list`` lists endpoint or collection roles
```

Comment:
The `**` syntax is asciidoc, right? I thought RST used indentation.

Reply:
_Holds head in hands_. Which non-markdown markdown language were we using again? 😂 

I think you're right. Indent + a single asterisk should be correct.